packages/backend/src/videos/videos.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { VideosService } from './videos.service';
+// import VideosDTO from '@project/common'
+@Controller('videos')
+export class VideosController {
+    constructor(private videosService: VideosService) {}
+
+    @Get()
+    showAllVideos(){
```

Comment:
brak mi tu typowania. Zdał by sie jakiś interfejs współdzielony z mobile i front - po to by typy pilnowaly nam różnic miedzy wysylanymi danymi a spodziewanymi przez aplikacje klienckie